dotenv-linter/Cargo.toml

```diff
@@ -0,0 +1,57 @@
+[package]
+name = "dotenv-linter"
+version.workspace = true
```

Comment:
Do you want to keep the same version for these crates? I don't mind, just wondering, because I guess `dotenv-lookup` would require much less updates then `dotenv-linter`, so it will publish not-changed crate for each release 🤔

Reply:
Yes, agree. Let's start from `v1.0.0`. What do you think?

Reply:
Make sense. yes